packages/backend/src/open_payments/wallet_address/key/routes.test.ts

```diff
@@ -28,13 +28,13 @@ describe('Wallet Address Keys Routes', (): void => {
     deps = await initIocContainer(Config)
     deps.bind('messageProducer', async () => mockMessageProducer)
```

Comment:
Does anybody know why we need this binding? That's why it doesn't build right now.

Reply:
@mkurapov @BlairCurrey @njlie @omertoast

Reply:
It looks like its safe to remove, it's not used anywhere